### ISSUE TITLE
‘R’ found 2 values on stack; needs 2

### ISSUE THREAD

**Issue by mccartney** [susam/fxyt#9]
```
N0N0XN100>[Y]R
```
yields a weird error message:
```
ERROR: (0, 0): 14: ‘R’ found 2 values on stack; needs 2
```

**Comment by mccartney** [comment]
I think it needs to say `3` here instead of `2`:
https://github.com/susam/fxyt/blob/19040eb282dc6470ec781d11418bb42154fad2a4/fxyt.html#L510
(and in other places)

**Comment by susam** [comment]
@mccartney Thank you for reporting this issue. This is now fixed in commit d90e920. Demo: https://susam.net/fxyt.html#N0N0XN100gbYcR
